sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/AutoConfiguredOpenTelemetrySdk.java

```diff
@@ -44,7 +44,10 @@ static AutoConfiguredOpenTelemetrySdk create(
     return new AutoValue_AutoConfiguredOpenTelemetrySdk(sdk, resource, config);
   }
 
-  /** Returns the {@link OpenTelemetrySdk} that was auto-configured. */
+  /**
+   * Returns the {@link OpenTelemetrySdk} that was auto-configured, or {@code null} if the SDK has
+   * been disabled.
+   */
   public abstract OpenTelemetrySdk getOpenTelemetrySdk();
```

Comment:
This PR does not appear to have or add a code path that can return `null`, and after several hours of poking at the current code, that fact does not seem to have changed since. Is this a bug? or did the design get changed after the doc was written?

Reply:
Oops. The response of this method was originally nullable but [per this conversation](https://github.com/open-telemetry/opentelemetry-java/pull/4489#discussion_r879949618), we went in a different direction. Looks like the javadoc never got updated - sorry about the confusion.

I've opened #6109 to fix the comment.